libethereum/BlockChainSync.cpp

```diff
@@ -177,6 +177,12 @@ BlockChainSync::~BlockChainSync()
 	abortSync();
 }
 
+void BlockChainSync::onBlockImported(BlockHeader const& _info)
+{
+	string number = dev::toString(_info.number());
```

Comment:
Conversion to string is not needed, just `static_cast<unsigned>(_info.number())`